app/src/server/api/routers/clan.ts

```
@@ -1498,6 +1527,39 @@ export const fetchClan = async (client: DrizzleClient, clanId: string) => {
     },
     where: eq(clan.id, clanId),
   });
+  // If leader is missing, promote a co-leader first, then a regular member
+  if (!clanData.leader) {
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)